lib/SieveSemantics.php

```diff
@@ -175,150 +237,189 @@ protected function makeValue_($arg)
     /**
      * Convert an extension (test) commands parameters from XML to
      * a PHP array the {@see Semantics} class understands.
-     * @param  array(SimpleXMLElement) $parameters
+     *
+     * @param  \SimpleXMLElement[]|\SimpleXMLElement $parameters
      * @return array
      */
-    protected function makeArguments_($parameters)
+    protected function makeArguments($parameters): array
```

Comment:
How can `$parameter` be a SimpleXMLElement?

Reply:
Constructor of SieveSementics: `$this->arguments = $this->makeArguments($xml);`